Assets/Scripts/StrategyGame/Models/World/Tile.cs

```diff
@@ -60,6 +60,10 @@ public class Tile
         /// </summary>
         public int HealthImpact { get; internal set; }
 
+        public int X { get; internal set; }
```

Comment:
It kinda saddens me that we're not using the perfectly fine `Coordinate` class (or a `Vector2`, now that I think about it), but alright!